src/commonMain/kotlin/exchange.dydx.abacus/processor/wallet/account/SubaccountProcessor.kt

```diff
@@ -130,6 +131,22 @@ internal open class SubaccountProcessor(
             payload = positions,
         )
 
+        val transfers: List<IndexerTransferResponseObject>?
+        val transferList = parser.asTypedList<IndexerTransferResponseObject>(content["transfers"])
+        if (transferList != null) {
+            transfers = transferList
+        } else {
+            val transfer = parser.asTypedObject<IndexerTransferResponseObject>(content["transfers"])
+            transfers = if (transfer != null) listOf(transfer) else null
```

Comment:
[q] wait, `content["transfers"]` can either a list of transfers for a singel transfer object? 👀 so weird

Reply:
Yeah.. the initial call potentially returns a list , and then the subsequent update is a single object. 
 There are integration tests depending on those.